alma/alma.php

```diff
@@ -117,7 +118,41 @@ public function __construct()
 
         $this->hook = new \Alma\PrestaShop\Helpers\HookHelper();
         $this->tabsHelper = new \Alma\PrestaShop\Helpers\Admin\TabsHelper();
-        $this->adminInsuranceHelper = new \Alma\PrestaShop\Helpers\Admin\InsuranceHelper($this);
+
+        $this->handlePSModule();
+    }
+
+    /**
+     * @return void
+     */
+    public function handlePSModule()
+    {
+        if (
+            $this->checkCompatibilityPSModule()
+            && $this->container === null
+        ) {
+            $this->container = new \PrestaShop\ModuleLibServiceContainer\DependencyInjection\ServiceContainer(
+                $this->name,
+                $this->getLocalPath()
+            );
+        }
+    }
+
+    /**
+     * @return bool
+     */
+    public function checkCompatibilityPSModule()
+    {
+        $versionHelper = new \Alma\PrestaShop\Helpers\ToolsHelper();
```

Comment:
Maybe we can add the injection in the constructor ?

Reply:
Done